src/processors/universal.ts

```diff
@@ -34,8 +35,11 @@ class UniversalProcessor extends BaseOrderProcessor {
   private priorityQueue = new Set<string>();
   private queue = new Set<string>();
   private ordersMap = new Map<string, IncomingOrderContext>();
-
+  private readonly UNLOCK_BATCH_SIZE = 10;
```

Comment:
Move this constant to `UniversalProcessorParams` as `batchUnlockSize`. Validate it's value in the constractor (possible values: `[1-10]`)